packages/flex-dev-utils/src/table.ts

```diff
@@ -45,11 +45,11 @@ export const isRegularMatrix = (matrix: string[][]) => matrix
  */
 export const printArray = (header: string[], data: string[][]) => {
   if (!isRegularMatrix(data)) {
-    logger.error('Table rows are not all the same length; this may produce irregular tabular view.');
+    logger.warning('Table rows are not all the same length; this may produce irregular tabular view.');
   }
 
   if (header.length !== Object.keys(data[0]).length) {
-    logger.error('Header length does not match data row length; printing table without header');
+    logger.warning('Header length does not match data row length; printing table without header');
```

Comment:
can we add a `.` at the end here or remove it from the warning above? don't care as long as we're being consistent